src/utils/transaction.ts

```diff
@@ -671,39 +560,61 @@ export const buildCrossChainAction = async (
               decimals: fromAssetDecimals,
               logoURI: fromAssetIconUrl,
             },
+            toAsset: { decimals: toAssetDecimals },
             amount,
-            offer,
+            receiverAddress,
             accountType,
           },
         } = transactionBlock;
-
+        let transactions: any[] = [];
```

Comment:
we don't need to assert type here I guess

Reply:
no `any[]` please 🙏